front/lib/api/assistant/configuration/table_query.ts

```diff
@@ -68,15 +75,30 @@ export async function fetchTableQueryActionConfigurations({
       const tablesQueryConfigTables =
         groupedAgentTablesQueryConfigurationTables[c.id] ?? [];
 
+      const tables: TableDataSourceConfiguration[] =
+        tablesQueryConfigTables.map((table) => {
+          const { dataSourceView } = table;
+
+          const dataSourceViewId = dataSourceView
+            ? DataSourceViewResource.modelIdToSId({
```

Comment:
note: sid vs id is SOOOO confusing, we should do a whole refactor to make sure it's SID everywhere once and for all (I always check the backend code to know what it _really_ expect)

Reply:
Yeah we are getting there. FWIW if it's a string -> sId otherwise ModelId 🙃